<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>610</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDownload">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>109</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string> sell         </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>       x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>    c</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>221</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="labelAltToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFusToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelExaToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAlcToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelGCPToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChrToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelJewToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChaToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChiToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelScoToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBleToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRegToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDivToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelVaaToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBauToCao">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>211</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelCaoToAlt">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToFus">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToExa">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToAlc">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToGCP">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToChr">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToJew">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToCha">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToChi">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToSco">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToBle">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToReg">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToDiv">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToVaa">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCaoToBau">
       <property name="text">
        <string>0/0/0/0/0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonAlteration">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Alteration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToAlt">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxAltToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButtonFusing">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Fusing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToFus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxFusToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_27">
       <item>
        <widget class="QPushButton" name="pushButtonExalted">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Exalted</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToExa">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxExaToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <item>
        <widget class="QPushButton" name="pushButtonAlchemy">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Alchemy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToAlc">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxAlcToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_24">
       <item>
        <widget class="QPushButton" name="pushButtonGemcutters">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Gemcutter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToGCP">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxGCPToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QPushButton" name="pushButtonChromatic">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Chromatic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToChr">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxChrToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QPushButton" name="pushButtonJewellers">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Jewellers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToJew">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxJewToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QPushButton" name="pushButtonChance">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Chance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToCha">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxChaToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonChisel">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Chisel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToChi">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxChiToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButtonScouring">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Scouring</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToSco">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxScoToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QPushButton" name="pushButtonBlessed">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Blessed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToBle">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxBleToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <item>
        <widget class="QPushButton" name="pushButtonRegret">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Regret</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToReg">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxRegToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QPushButton" name="pushButtonDivine">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Divine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToDiv">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxDivToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_26">
       <item>
        <widget class="QPushButton" name="pushButtonVaal">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Vaal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToVaa">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxVaaToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_25">
       <item>
        <widget class="QPushButton" name="pushButtonBauble">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Bauble</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCaoToBau">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxBauToCao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
